Gemfile

```diff
@@ -34,6 +34,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'aasm', '>=5.0.1' 
```

Comment:
The gems in group `development` get installed only in the `development` environment. Once we go live this gem will not be available. I don't think that's what you wanted.